webook/arrangement/urls/event_serie_urls.py

```diff
@@ -1,8 +1,11 @@
+from django.db import router
 from django.urls import path
 from webook.arrangement.views import (
     event_serie_delete_file_view,
     delete_event_serie_view,
     event_serie_manifest_view,
+    calculate_event_serie_view,
+    calculate_event_serie_preview_view
```

Comment:
It is common to have imports in alphabetical order. Your editor should be setup to do this for you automatically.